app-arch/innoextract/innoextract-1.5.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils toolchain-funcs cmake-utils
+
+DESCRIPTION="A tool to unpack installers created by Inno Setup"
+HOMEPAGE="http://constexpr.org/innoextract/"
+SRC_URI="http://constexpr.org/innoextract/files/${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="c++0x debug doc +iconv +lzma static"
+
+RDEPEND="
+	!static? (
+		dev-libs/boost:=
+		iconv? ( virtual/libiconv )
+		lzma? ( app-arch/xz-utils )
+	)"
+DEPEND="${RDEPEND}
+	doc? ( >=app-doc/doxygen-1.8.3.1 )
+	static? (
+		app-arch/bzip2[static-libs]
+		dev-libs/boost[static-libs]
+		sys-libs/zlib[static-libs]
+		iconv? ( virtual/libiconv )
+		lzma? ( app-arch/xz-utils[static-libs] )
+	)"
+
+DOCS=( README.md CHANGELOG )
+
+pkg_pretend() {
+	if [[ ${MERGE_TYPE} != binary ]]; then
+		# not sure about minimum clang req
+		if use c++0x && [[ $(tc-getCXX) == *g++ && $(tc-getCXX) != *clang++ ]] ; then
+			if [[ $(gcc-major-version) == 4 && $(gcc-minor-version) -lt 7 || $(gcc-major-version) -lt 4 ]] ; then
+				eerror "You need at least sys-devel/gcc-4.7.0 for C++0x capabilities"
+				die "You need at least sys-devel/gcc-4.7.0 for C++0x capabilities"
+			fi
+		fi
+	fi
+}
+
+src_configure() {
+	local mycmakeargs=(
+		$(usex iconv -DWITH_CONV=iconv -DWITH_CONV=builtin)
+		$(cmake-utils_use_use lzma LZMA)
```

Comment:
`cmake-utils_use_use` uses `ON` and `OFF` - it might also work with `yes` and `no` (or `usex lzma ON OFF` would work), but I think `cmake-utils_use_use` is more idiomatic when using `cmake-utils`